### ISSUE TITLE
More user friendly interface of settings

### ISSUE THREAD

**Issue by llybin** [btdex/btdex#36]
I also read in Discord a suggestions about it.

Possible misunderstanding with icons lock and exit. Most don't will wait displaying hint. But behavior is not expected.

![image](https://user-images.githubusercontent.com/5445357/72611969-0ca3ec80-395e-11ea-9795-c063512bdb53.png)

I propose create button settings, and move there most of settings. Choosing explorer, node and etc. On main window display only connection status to node green/red icon foe example.
And give more meaningful names to the buttons, in settings window will space for it.

**Comment by andylamgot** [comment]
![image](https://user-images.githubusercontent.com/37191442/72660908-3ff28400-3a0f-11ea-9ea2-1e53c5b82f0d.png)
Trade button(buy/sell) can be more significant, and move to better position.
Add a title or section for each Buy / Sell Order, make it more clear. 
with market depth chart will be a must in next phrase...

**Comment by leetlezee** [comment]
I would also suggest rearranging the top part of the window... currently, left to right, you have market, Burst address, Burst balance, and then your balance in the selected market. I would either move the burst address and balance to the far left or right, and group the market and your balance next to each other.

E.g. layout like following:

TRT (market dropdown) / 100 TRT balance / 3000 Burst balance / BURS-TADD-RESS-HERE0

**Comment by jjos2372** [comment]
Thanks for the suggestions. Just take into account that for cross-chain/fiat markets you would not see your balance on that specific market (BTC, LTC, USD, etc.) but TRT (to monitor your rewards). That is part of the reasoning for the current setup.

We could also try to move the market selection dropdown to the market TAB itself, but then the same would apply to the history panel. Unless we marge the history with market on a single TAB (as most exchanges do these days).

**Comment by shefass** [comment]
I agree with Lev, Clear pin and Signout buttons can be moved to Settings.

2020-07-13, pr, 22:40 jjos2372 <notifications@github.com> rašė:

**Comment by goodplanets** [comment]
![image](https://user-images.githubusercontent.com/40472463/89739102-6e838b00-da43-11ea-8708-d4b5e38d82a3.png)
the scroll bar does not get exact numbers, maybe a field where exact numbers can be entered so the user doesn't have to try to play with the scroll bar for fine adjustment?
